### PR TITLE
docs: Add note to local setup for WSL users to skip mongodb creation step.

### LINKED ISSUE #47493: Article on how to setup freeCodeCamp locally inadvertently gives mongodb server starting instructions twice, leading to confusion
### Describe the Issue

Windows users attempting to follow each step outlined in the article https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally will encounter an error on the 2nd mongoDB server startup step because it was already started in an earlier instruction.

The first time mongoDB server is started is in the steps linked from "How to prepare your local machine". The second time is later on in the article in the section called "Step 3: Start MongoDB and seed the database".

The error encountered the second time is:
Command 'mongod' not found, but can be installed with:
sudo apt install mongodb-server-core

Users may not realize that they had already done all they needed to do for mongodb setup in a prior step leading to confusion.

### Affected Page

https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally

### Steps to Reproduce

on Windows 10+, follow all the instructions in the article (https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally) including the ones mentioned in (https://contribute.freecodecamp.org/#/how-to-setup-wsl) which is linked from the first article.

Further Details:
Readers encounter the first section "How to prepare your local machine" where it says: "We also support Windows 10 via WSL2, which you can prepare by [reading this guide](https://contribute.freecodecamp.org/#/how-to-setup-wsl)." 
The link given goes to an article "Set up freeCodeCamp on Windows Subsystem for Linux (WSL)" and has multiple steps within, which include "Start the MongoDB service at port 27017..." with accompanying instructions.

Once finished following the steps provided in the second article, we return back to the first reference article to continue the remaining steps following the section "How to prepare.." And as we do so, we encounter again the instruction to start the MongoDB (Step 3: Start MongoDB and see the database). Blindly following this step then we see this error:
"Command 'mongod' not found, but can be installed with:
sudo apt install mongodb-server-core"

And there arises the confusion. Didn't we install it already? Maybe this is something else? So we go ahead and install it again. Leading to further errors down this road such as the one mentioned in issue #45784

### Recommended fix or suggestions

Clarification needs to be given to explain that the 2nd time we are told to start it with mongod command is only needed if it hadn't already been started successfully before (perhaps with a suggestion to check the service mongodb://localhost:27017 first).

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by ppeters0502**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47493

This change adds the following note to how-to-setup-freecodecamp-locally.md since users who follow the WSL setup guide already have a MongoDB instance setup by the time they get to the MongoDB creation step.

> [!NOTE]
> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.

screenshot-freecodecamp

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -213,6 +213,9 @@ Before you can run the application locally, you will need to start the MongoDB s
 > [!NOTE]
 > Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
 
+> [!NOTE]
+> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.
+
```
Comment (naomi-lgbt) [inline docs/how-to-setup-freecodecamp-locally.md:218]:
```suggestion
> [!NOTE]
> Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
>
> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.
```

Two note-boxes together looks off - can we combine them?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/how-to-setup-freecodecamp-locally.md (hunk 1 above)
Comment (ppeters0502) [reply docs/how-to-setup-freecodecamp-locally.md]: I made the requested change, here's a quick screenshot of how it looks
screenshot-freecodecamp_updated

**Review by naomi-lgbt — DISMISSED** [review]
banger pr, thanks for contributing!

**Comment by ppeters0502** [comment]
@naomi-lgbt My apologies, I didn't check with the linter when I committed your requested change earlier. Not sure if you have to re-review since I pushed more changes, but if you could re-review, that would be great!

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
>  I didn't check with the linter when I committed your requested change earlier. 

Haha no worries - my fault for giving a borked suggestion

**Review by Sembauke — APPROVED** [review]

**Pull request merged**